lib/modules/subredditManager.js

```diff
@@ -516,6 +519,14 @@ function editSubredditShortcut(ele, event) {
 	`;
 	$editShortcutDialog.html(thisForm);
 
+	$editShortcutDialog.find('#shortcut-subreddit').on('keyup', _.throttle(({ currentTarget }) => {
```

Comment:
`{ currentTarget }: KeyboardEvent` will placate flow; you can also do the renaming in destructuring if you want: `{ currentTarget: shortcut }: KeyboardEvent`